nix/vast/default.nix

```diff
@@ -17,6 +17,7 @@
 , python3
 , jq
 , tcpdump
+, utillinux
```

Comment:
What's this?

Reply:
A small fix that enables colored log output in the Nix builds. We should cherry-pick it into master.

Reply:
Yes please.